### ISSUE TITLE
from_design() method not working with sample code

### ISSUE THREAD

**Issue by tagawa** [elevenlabs/elevenlabs-python#94]
Using the [sample code](https://github.com/elevenlabs/elevenlabs-python/blob/main/API.md#voicedesign) in the docs, the [`from_design()` method](https://github.com/elevenlabs/elevenlabs-python/blob/3d1f80a3f33a76c9b84054313df5e3e5f5a96726/elevenlabs/api/voice.py#L118) is returning the following error. It looks like the method's API call is responding with a list whereas a dictionary or JSON object is expected.

```
Traceback (most recent call last):
  File "/Users/daniel/Documents/python/elevenlabs/save-voice.py", line 25, in <module>
    voice = elevenlabs.Voice.from_design(design)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/daniel/Library/Python/3.11/lib/python/site-packages/elevenlabs/api/voice.py", line 128, in from_design
    response = API.post(url, json=data)
               ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/daniel/Library/Python/3.11/lib/python/site-packages/elevenlabs/api/base.py", line 65, in post
    return API.request(url, method="post", *args, **kwargs)  # type: ignore
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/daniel/Library/Python/3.11/lib/python/site-packages/elevenlabs/api/base.py", line 46, in request
    error = HTTPError(response)
            ^^^^^^^^^^^^^^^^^^^
  File "/Users/daniel/Library/Python/3.11/lib/python/site-packages/elevenlabs/api/error.py", line 14, in __init__
    self.message = detail["message"]
                   ~~~~~~^^^^^^^^^^^
TypeError: list indices must be integers or slices, not str
```

**Comment by flavioschneider** [comment]
Hi, this issue has been fixed in `v0.2.25`, thanks!

**Comment by tagawa** [comment]
Great, thank you @flavioschneider.
